Libraries/Components/DatePicker/DatePickerIOS.ios.js

```diff
@@ -154,7 +159,7 @@ class DatePickerIOS extends React.Component<Props> {
                 ? props.initialDate.getTime()
                 : undefined
           }
-          locale={props.locale ? props.locale : undefined}
+          locale={props.locale}
```

Comment:
I don't think this is the same behavior. React handles null (allowed by the flow type) differently than undefined. Was there a flow error that required this change?

Reply:
Yes... I got this error...
```
Sketchy null check on string [1] which is potentially an empty string. Perhaps
you meant to check for null or undefined [2]? (sketchy-null-string)
```

I think this is a problem of flow. The situation is simlilar to this issue https://github.com/facebook/flow/issues/4608

Reply:
Ah, this should be changed to the following then to be semantically the same. 

```
locale={(props.locale != null && props.locale != "")  ? props.locale : undefined}
```